src/bscPlugin/hover/HoverProcessor.ts

```diff
@@ -158,7 +158,7 @@ export class HoverProcessor {
                 const fullName = processedTypeChain.fullNameOfItem || token.text;
                 const useCustomTypeHover = isInTypeExpression || expression?.findAncestor(isNewExpression);
                 let hoverContent = fence(`${fullName} as ${exprType.toString()}`);
-                if (isFunctionType(exprType)) {
+                if (isTypedFunctionType(exprType)) {
```

Comment:
Should this be `isCallableType`?

Reply:
this one should be TypedFunctionType, because only typed functions have docs and stuff